### PR TITLE
Bump version to v1.0.6

### PR DESCRIPTION
**Pull request by BradleyBoutcher**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [1.0.6] - 2020-07-01
+
 ### Added
 - Plugin support authenticating with access token provided by authn-k8s.
```
Comment (izgeri) [inline CHANGELOG.md:12]:
```suggestion
- Plugin supports authenticating with Conjur access token (for example, if provided by authn-k8s).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -34,5 +36,6 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Added support to configure the use of the plugin via environment variables
 
 [Unreleased]: https://github.com/cyberark/ansible-conjur-collection/compare/v1.0.5...HEAD
```
Comment (diverdane) [inline CHANGELOG.md:38]: I think this needs to be:
```
[Unreleased]: https://github.com/cyberark/ansible-conjur-collection/compare/v1.0.6...HEAD
```

File: CHANGELOG.md (hunk 1 above)
Comment (BradleyBoutcher) [reply CHANGELOG.md]: Suggestion committed 👍

File: CHANGELOG.md (hunk 2 above)
Comment (BradleyBoutcher) [reply CHANGELOG.md]: Updated release number there 👍

**Pull request merged**